transport/nio/src/main/java/org/kaazing/gateway/transport/nio/internal/AbstractNioAcceptor.java

```diff
@@ -377,23 +377,19 @@ public UnbindFuture unbind(final ResourceAddress address) {
         // a hang when there's a race with Gateway.destroy().
         if (!unbindScheduler.isShutdown()) {
```

Comment:
If this is not scheduled onto unbindScheduler, no point in checking for shutdown status